### PR TITLE
chebyshev_filter_oder_from_distortion_and_frequencies

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests

### CODE REVIEW

File: symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless, convert_to_float)
+
+# Description
+## The approximation of the power transmission coefficient of a normalized low-pass filter is given by approximating functions of the order of n.
+## The Chebyshev filter is described by the function 1 / cos(n * acos(1 / frequency)).
+## Bandwidth distortion determines the maximum distortion in the bandwidth. In other words, to determine the level of ripples in the bandwidth.
+## The band-stop distortion sets the required suppression level in the filter band-stop.
+
+## Law is: N = acos(es / ep) / acos(fs / fp), where
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py:12]: Why do you have different formula for exactly the same description of the filter?

Comment (Zulusssss) [reply symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py:12]: > Why do you have different formula for exactly the same description of the filter?

This is a low-pass filter. In another law, there is a high-pass filter. But both of these filters are based on the same polynomial.

Should I add an explanation about which frequencies the filter filters?

Comment (blackyblack) [reply symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py:12]: Of course

File: symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 ## The Chebyshev filter is described by the function 1 / cos(n * acos(1 / frequency)).
 ## Bandwidth distortion determines the maximum distortion in the bandwidth. In other words, to determine the level of ripples in the bandwidth.
 ## The band-stop distortion sets the required suppression level in the filter band-stop.
+## In this case, a low-pass filter is considered. The low-pass filter passes all frequencies from 0 to the set frequency.
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/low_pass_chebyshev_filter_order_from_distortion_and_frequencies.py:11]: It should also be reflected in filter symbol name.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**